### PR TITLE
Creating a production playbook for the research data machines

### PR DESCRIPTION
**Pull request by carolyncole**
Also updated postgres to get around unknown attribute login_port

### CODE REVIEW

File: roles/pulibrary.postgresql/tasks/create_db.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   postgresql_db:
     name: '{{ application_db_name }}'
     login_host: '{{ postgres_host | default(omit) }}'
-    login_port: '{{ postgres_port | default(omit) }}'
+    port: '{{ postgres_port | default(omit) }}'
```
Comment (carolyncole) [inline roles/pulibrary.postgresql/tasks/create_db.yml:6]: @kayiwa this change was really strange, but I got a no attribute error when running the playbook.

File: roles/pulibrary.postgresql/tasks/create_users.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   postgresql_user:
     name: '{{ application_dbuser_name }}'
     login_host: '{{ postgres_host | default(omit) }}'
-    login_port: '{{ postgres_port | default(omit) }}'
+    port: '{{ postgres_port | default(omit) }}'
```
Comment (carolyncole) [inline roles/pulibrary.postgresql/tasks/create_users.yml:6]: @kayiwa here too.  Not entirely sure why...

File: roles/pulibrary.postgresql/tasks/create_db.yml (hunk 1 above)
Comment (kayiwa) [reply roles/pulibrary.postgresql/tasks/create_db.yml:6]: surprised it worked given this. https://docs.ansible.com/ansible/2.3/postgresql_db_module.html (there's no `login_port`) Good catch.

File: roles/pulibrary.postgresql/tasks/create_users.yml (hunk 2 above)
Comment (kayiwa) [reply roles/pulibrary.postgresql/tasks/create_users.yml:6]: same here. https://docs.ansible.com/ansible/latest/modules/postgresql_user_module.html

**Pull request merged**